daemon/daemon.go

```diff
@@ -980,6 +985,10 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	return controller, nil
 }
 
+func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Its a circular dependency issue (server depends on daemon).
I faced with the exact same dilemma : Should we expose networkController to api/server (or) import net/http in daemon. I chose the later because exposing networkController outside of daemon is worse :-)